lib/OpenQA/Scheduler/Scheduler.pm

```diff
@@ -669,6 +669,44 @@ sub _job_stop_children {
     }
 }
 
+sub _carry_over_labels {
+    my ($job) = @_;
+    if ($job->comments > 0) {
```

Comment:
Is it possible to hit this condition? When job is set as !PASSED then comments are carried over, right? So without some good chunk of timing luck, you don't even have chance to add any new comments. Or did I miss something?